### DISCUSSION TITLE
OpenAM 14.6.4 still contains the CVE-2021-29156

### DISCUSSION THREAD

**Discussion by hoangnguyen115** [OpenIdentityPlatform/OpenAM#459] [Q&A] [answered]
Hi guys,

I am still able to exploit the amadmin password hash (the CVE-2021-29156) after the upgrade of OpenAM to the version 14.6.4 (following this article: https://portswigger.net/research/hidden-oauth-attack-vectors). As it is stated that is CVE has been fixed since 14.6.3, can anyone explain me why the exploit still works?

Thanks

**Comment by vharseko** [comment]
@maximthomas

**Comment by maximthomas** [comment, marked as answer]
@hoangnguyen115 The issue was fixed but has was not been released yet

**Reply by hoangnguyen115** [reply]
Thank you @maximthomas and @vharseko
